libs/api/domains/notifications/src/lib/notifications.model.ts

```diff
@@ -29,20 +19,20 @@ export class NotificationMetadata {
   @Field(() => GraphQLISODateTime, { nullable: true })
   created?: Date
 
-  @Field(() => GraphQLISODateTime, { nullable: true })
-  read?: Date
+  @Field(() => Boolean, { nullable: true })
```

Comment:
Don't have to explicitly declare boolean here
![image](https://github.com/island-is/island.is/assets/9449413/bed0399b-7237-42f8-97d5-ab118584f23d)

Reply:
omg i'll remove it.